From 97334815c4a7cf0911b30c1366bbe67e883c57dd Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Marc-Andr=C3=A9=20Lureau?= <marcandre.lureau@redhat.com>
Date: Wed, 5 Jun 2024 10:28:20 +0400
Subject: [PATCH 4/4] rhel 9.4.0 machine type compat for virtio-gpu migration
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

RH-Author: Marc-André Lureau <marcandre.lureau@redhat.com>
RH-MergeRequest: 246: virtio-gpu: fix v2 migration
RH-Jira: RHEL-34621
RH-Acked-by: Peter Xu <peterx@redhat.com>
RH-Acked-by: Thomas Huth <thuth@redhat.com>
RH-Commit: [2/2] 2a895d510453e83bb45dfb39b7751bcc2f309cc5 (marcandre.lureau-rh/qemu-kvm-centos)

Signed-off-by: Marc-André Lureau <marcandre.lureau@redhat.com>
---
 hw/core/machine.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/hw/core/machine.c b/hw/core/machine.c
index cf1d7faaaf..92609aae27 100644
--- a/hw/core/machine.c
+++ b/hw/core/machine.c
@@ -310,6 +310,8 @@ GlobalProperty hw_compat_rhel_9_5[] = {
     { TYPE_VIRTIO_IOMMU_PCI, "granule", "4k" },
   /* hw_compat_rhel_9_5 from hw_compat_8_2 */
     { TYPE_VIRTIO_IOMMU_PCI, "aw-bits", "64" },
+  /* hw_compat_rhel_9_5 from hw_compat_8_2 */
+    { "virtio-gpu-device", "x-scanout-vmstate-version", "1" },
 };
 const size_t hw_compat_rhel_9_5_len = G_N_ELEMENTS(hw_compat_rhel_9_5);
 
-- 
2.39.3

